src/sentry/mediators/sentry_app_installations/notifier.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import absolute_import
+
+from sentry.http import safe_urlopen, safe_urlread
+from sentry.mediators import Mediator, Param
+from sentry.utils.cache import memoize
+
+
+class Notifier(Mediator):
+    install = Param('sentry.models.SentryAppInstallation')
+
+    def call(self):
+        self._send_webhook()
+
+    def _send_webhook(self):
+        safe_urlread(
+            safe_urlopen(self.sentry_app.webhook_url, json=self.body, timeout=5)
+        )
+
+    @property
+    def body(self):
+        return {
+            'sentry_app': {
+                'uuid': self.sentry_app.uuid,
+                'slug': self.sentry_app.slug,
+            },
```

Comment:
Should we use the API serializers here? It would be good to have consistency between the API representations and representations used in webhooks.

Reply:
Yea, we should. I'm going to change this to be more in the shape of a general event payload (ie. have `event: installation`, etc.) anyway, so it'd be good to encapsulate that in a serializer 👍